src/utilities/paginate.test.ts

```diff
@@ -14,7 +14,7 @@ vi.mock('../models/ctype', () => ({
     findAll: vi.fn(),
     count: vi.fn(),
   },
-  groupForAttestationsCount: [],
+  groupForAttestationsCreated: [],
```

Comment:
Oh, very nice to be reminded of this line! When our attestations count was a computed field, we had to use `group by` in our SQL, and this required jumping through some hoops. All these could and should be removed now!

Reply:
I'm not sure what you mean by "all these".

Do you mean only this kind of line on each test?

Reply:
I expected there would be a few more mentions of "group" in the project, but looks like they were removed previously. Even this line can be removed now, as far as I can see.